packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.types.ts

```diff
@@ -54,7 +54,7 @@ export interface IDetailsItemProps {
   cellStyleProps?: ICellStyleProps;
 }
 
-export interface IDetailsRowBaseProps extends IBaseProps<IDetailsRow>, IDetailsItemProps {
+export interface IDetailsRowBaseProps extends Pick<IDetailsListProps, 'onRenderItemColumn'>, IBaseProps<IDetailsRow>, IDetailsItemProps {
```

Comment:
need @natalieethell 's work to document all these inherited props ^_^

Reply:
Agreed

Reply:
I think api-extractor 7 would just generate something like 

```tsx
interface IDetailsRowBaseProps extends Pick<IDetailsListProps, 'onRenderItemColumn'>, IBaseProps<IDetailsRow>, IDetailsItemProps {
  ...
}
```

in its api.ts file. We could, as a start, link to IDetailsListProps, and then maybe in the next iteration just document those props directly in IDetailsRowBaseProps. Not sure how much work that would be yet, but it's a thought!